Sources/TensorFlow/Operators/Basic.swift

```diff
@@ -1229,7 +1229,7 @@ extension Tensor {
     /// Returns `true` if all given axes are in the range `[-rank, rank)`.
     @usableFromInline
     internal func areAxesInRange<T: BinaryInteger>(_ axes: [T]) -> Bool {
-        return !axes.contains(where: { !isAxisInRange($0) })
+        return !axes.contains(where: { !isAxisInRange($0) }) && ensureNonDuplicated(axes)
```

Comment:
This line's change appears to give the method new semantics that clash with its name and its doc comment

Reply:
That's an excellent point. Given that the initial purpose of this PR was to improve the error messages, I've removed this functionality in this PR.